_licenses/cddl-1.0.txt

```diff
@@ -0,0 +1,378 @@
+---
+title: Common Development and Distribution License 1.0
+spdx-id: CDDL-1.0
+
+description:
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+using:
+  - OpenZFS: http://open-zfs.org
+  - NetBeans: https://netbeans.org/
+  - GlassFish: https://github.com/eclipse-ee4j/glassfish
+
+permissions:
```

Comment:
I believe this should be:

* `commercial-use`
* `modification`
* `distribution`
* `private-use`
* `patent-use`